MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ComboBox.xaml

```diff
@@ -910,131 +910,18 @@
     <Setter Property="wpf:TextFieldAssist.TextFieldCornerRadius" Value="4,4,0,0" />
     <Style.Triggers>
       <Trigger Property="IsMouseOver" Value="True">
-        <Setter Property="Background" Value="{DynamicResource MaterialDesignTextFieldBoxHoverBackground}" />
+        <Setter Property="Background" Value="{DynamicResource MaterialDesign.Brush.TextBox.HoverBackground}" />
       </Trigger>
     </Style.Triggers>
   </Style>
 
   <Style x:Key="MaterialDesignOutlinedComboBox"
          TargetType="ComboBox"
          BasedOn="{StaticResource MaterialDesignFloatingHintComboBox}">
-    <Setter Property="BorderBrush" Value="{DynamicResource MaterialDesignTextAreaBorder}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource MaterialDesign.Brush.Header.Foreground}" />
```

Comment:
Though it looks a little off I think it is the correct brush. This matches the border for the text box and since a combo box is just a loose wrapper on that, and this is easily overridable I think it is fine.